### PR TITLE
docs: update pushing-an-image-to-kind.md - add info re: crictl

### PR DESCRIPTION
**Pull request by tmancill**
### What does this PR do?

Update the documentation regarding listing images pushed to Kind.

### How to test this PR?

(N/A)

### CODE REVIEW

File: website/docs/kubernetes/kind/pushing-an-image-to-kind.md [hunk 1]
```diff
@@ -26,7 +26,15 @@ With Podman Desktop, you can push an image to your local Kind-powered Kubernetes
...
-Kind does not enable you to list loaded images.
+With recent versions of Kind, the `crictl` command can be used - e.g., `podman exec -it kind-cluster-control-plane crictl images`.  The name of the control plane container may vary, so you can use a filter to query for the container:
+
+```
+podman exec -it $(podman ps --filter "label=io.x-k8s.kind.role=control-plane" --format {{.Names}}) crictl images
+```
+
+See the [Kind Quickstart](https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster) for details.
+
+Older versions of Kind do not allow you to list loaded images.
```
Comment (benoitf) [inline website/docs/kubernetes/kind/pushing-an-image-to-kind.md]:
```suggestion
```

I think we can remove what was in the past

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
@tmancill looks like one file is not formatted (need to run `yarn` and then `yarn format:fix`)

**Comment by benoitf** [comment]
if you can't I'll apply it in your branch

**Comment by tmancill** [comment]
> if you can't I'll apply it in your branch

I will be able to, but not right away.  Feel free to apply it if it's easy for you.  (And thank you!)

**Comment by tmancill** [comment]
Closing in favor of #6666.  (Apologies for the extra noise @benoitf .)

**Pull request closed without merging**